PeerioTechnologies/peerio-desktop
# chore: PeerUI CustomIcon no longer needs individual icon SVGs

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/16962/peerui-make-customicon-svgs-inline
PeerUI: https://github.com/PeerioTechnologies/ui-library/pull/19
MC: https://github.com/PeerioTechnologies/medcryptor-desktop/pull/4

#### Testing instructions
This is a behind-the-scenes update to CustomIcon that shouldn't affect how they look, so all custom icons should look the same as they do on prod. They are used in the following places:
* main AppNav menu
* "pin" icon at the top of a DM
* "pin" icon at the left of a DM entry in the chat list, if that DM is a "favorite"
* "leave" icon in the sidebar of a room (if you are not an admin)
* "move" icon in the file actions and folder actions menus
* "remove-member" icon in the file actions and shared folder actions menus
* "shared folder" icon in the Files view
* "stacked files" icon in the Files view, if there's a folder currently being converted to shared folder

NOTE: needs a PeerUI release and branch update before merge

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
